docs/buildah-inspect.md

```diff
@@ -0,0 +1,27 @@
+## buildah-mount "1" "May 2017" "buildah"
+
+## NAME
+buildah inspect - Display information about a working container.
+
+## SYNOPSIS
+**buildah** **inspect** [*options* [...] --] **containerID**
+
+## DESCRIPTION
+Prints information about a working container's configuration.
+
+## OPTIONS
+
+**--format** *template*
+
+Use *template* as a Go template when formatting the output.
```

Comment:
The description now includes:

Users of this option should be familiar with the [*text/template* package](https://golang.org/pkg/text/template/) in the Go standard library, and of internals of buildah's implementation.

If we want to go bigger, we can, but I don't have anything in mind.